index.html

```diff
@@ -26,6 +26,20 @@ <h1 id="logo">Aziz Mejri</h1>
           </li>
         </ul>
       </nav>
+
+      <div id="mobile-nav" class="invisible">
+        <div>
+          <i id="close" class="fas fa-xmark"></i>
+          <div>
+            <li><a class="mobile-link" href="#projects">Portoflio</a></li>
+            <li><a class="mobile-link" href="#about-m">About</a></li>
```

Comment:
- [ ] Kindly, change the value of the `href` attribute from #about-m' to '#about-me' because it'll direct it to the correct part of the page.